### DISCUSSION TITLE
Can you customize the email domain from Clerk when a user signs up via email?

### DISCUSSION THREAD

**Discussion by lancejpollard** [clerk/javascript#3077] [Support request - DO NOT OPEN HERE]
The "Domains > DNS" page on the Clerk dashboard has 3 email domains it wants me to add CNAMEs for, like `clk._domainkey.mysite.com`. Does that mean the emails will come from something like `no-reply@clk._domainkey.mysite.com`? Or how do the email domains show up? I couldn't find that info anywhere in the docs.

Is it possible to have the emails come from a specific domain (`mysite.com`) or subdomain (`mail.mysite.com`)?

Thanks!

**Comment by princealireh** [comment]
@lancejpollard were you able to get a solution to this?

**Comment by a-fatah** [comment]
After configuring CNAME records, clerk will be able to send emails using notifications@yourdomain.com as FROM address. You can change the FROM email address using backend api

**Comment by jescalan** [comment]
https://github.com/clerk/javascript/discussions/3829

**Reply by amacks** [reply]
the referenced discussion seems to have been deleted.

**Reply by jescalan** [reply]
Sorry about that, just updated!
